server/models/ClubProfileModel.js

```diff
@@ -3,6 +3,7 @@ const Schema = mongoose.Schema;
 
 const clubProfileSchema = new Schema({
     name: { type: String, required: true },
+    clubAccountId: {type: String, required: true},
```

Comment:
in your implementation of the `create` function, you created an object using only `req.body` first and then added the `clubAccountId` field later; if you plan to do that, then the `clubAccountId` field cannot be required (otherwise we can't create a club account without a club account id). However, as we discussed during the meeting, we do want this field to be required. Therefore, maybe consider changing the flow of the `create` function so that you create a js object out of the given `req.body` that actually contains the account id (which is only in `req.params.accountId`, not in `req.body`)? In this case, we can create a new object directly using the generated js object that includes account id, which can then stay as a required field.